### ISSUE TITLE
Increase Accessibility for Keyboard, Screen Reader, and iOS/VoiceOver users

### ISSUE THREAD

**Issue by foliovision** [foliovision/freedom-video-player#1048]
Original issue link: [#1453 Increase Accessibility for Keyboard, Screen Reader, and iOS/VoiceOver users](https://github.com/flowplayer/flowplayer/issues/1453)

----

<img width='32' height='32' src='https://avatars.githubusercontent.com/u/1063312?v=4' /> [dovematrix](https://github.com/dovematrix) commented on Nov 14, 2019

Could you please increase the accessibility of the FlowPlayer?  My university has purchased a commercial license 7.2.4.  Keyboard and screen-reader users on desktop cannot play video since it doesn’t appear to receive any focus. iOS/VoiceOver users also cannot play the video.

----

<img width='32' height='32' src='https://avatars.githubusercontent.com/u/265438?v=4' /> [nnarhinen](https://github.com/nnarhinen) commented on Nov 15, 2019

Hello. Flowplayer7 will not receive any new features anymore. For better accessibility you should take a look at our latest player called "Flowplayer Native". Accessibility documentation for that can be found from https://flowplayer.com/developers/player/accessibility
